resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolverBuilder.java

```diff
@@ -201,6 +202,16 @@ public DnsNameResolverBuilder authoritativeDnsServerCache(AuthoritativeDnsServer
         return this;
     }
 
+    /**
+     * Sets the udp client listen bindAddress
+     * @param bindAddress
+     * @return {@code this}
+     */
+    public DnsNameResolverBuilder bindAddress(String bindAddress) {
```

Comment:
please call this `localAddress` to be consistent with `Bootstrap` and make it take a `SocketAddress`.

Reply:
Rename the name to `localAddress`, And change the type to `SocketAddress `